Makefile

```diff
@@ -57,7 +57,7 @@ STAT           = stat -c '%Y %n'
 endif
 
 TYPES_FILES    = $(shell find pkg/apis -name types.go)
-GO_VERSION    ?= 1.11.3
+GO_VERSION    ?= 1.11.5
```

Comment:
the other option here is to go back to the unanchored version "1.11" at which point docker will always be pulling the freshest "1.11"